csharp/test/Slice/escape/Clash.ice

```diff
@@ -24,9 +24,10 @@ interface Intf
 
     void op(string context, string current, string response, string ex, string sent, string cookie,
             string sync, string result, string istr, string ostr, tag(1) string? proxy, tag(2) int? obj);
-    void opOut(out string context, out string current, out string response, out string ex,
-               out string sent, out string cookie, out string sync, out string result, out string istr,
-               out string ostr, out tag(1) string? proxy, out tag(2) int? obj);
+    (string context, string current, string response,
```

Comment:
Indentation here looks bogus and the changes is unrelated to removing exceptions specifications